src/overlay/PeerBareAddress.h

```diff
@@ -66,5 +66,6 @@ class PeerBareAddress
     Type mType;
```

Comment:
all members should be const now

Reply:
Nope. Tried. We use it in too many places where a value of type PeerBareAddress gets assigned. C++ const is mostly useless in fields of value-like objects, only non-copied / non-moved objects.